rbuilder/Dockerfile

```diff
@@ -3,22 +3,23 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get --no-install-recommends -y install \
     pciutils build-essential git wget \
     lsb-release dpkg-dev curl bsdmainutils fakeroot
-RUN mkdir -p /usr/local/share/cosmos-sdk/
+RUN mkdir -p /usr/local/share/tendermint/
```

Comment:
is this image specific to tendermint? I should it was used by the sdk as well?

Reply:
This image is for tendermint and cosmos-sdk. The platform agnostic files should reside somewhere in `/usr/{,local}/share/PROGDIR`. How about `/usr/local/share/rbuilder/`?